Author: GeorgeClark <GeorgeClark@0b4bb1d4-4e5a-0410-9cc4-b2b747904278>
Date:   Wed Feb 13 00:49:20 2012 +0000

    Item12391: More validations   This patch applies to Foswiki 1.1.0-1.1.2 Post Item12285 patch

~~~PATCH 5218b3501c96c8e6017f4df81298ef5b:e89b823a46626febb00fe9f0f26ff332 lib/Foswiki/Macros/MAKETEXT.pm (Foswiki-1.1.0,Foswiki-1.1.1,Foswiki-1.1.2)
--- lib/Foswiki/Macros/MAKETEXT.pm	2013-02-13 11:00:18.037967200 -0500
+++ lib/Foswiki/Macros/MAKETEXT.pm	2013-02-13 11:00:39.645714522 -0500
@@ -25,8 +25,8 @@
     $str =~ s/\]/~]/g;
 
     # restore already escaped stuff:
-    $str =~ s/~~\[/~[/g;
-    $str =~ s/~~\]/~]/g;
+    $str =~ s/~~+\[/~[/g;
+    $str =~ s/~~+\]/~]/g;
 
     $max         = 0;
     $min         = 1;

